packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -79,7 +79,7 @@ class FluentLocalizer {
 
     const subject = await l10n.formatValue(mailSubject);
     return {
-      html: document.documentElement.outerHTML,
+      html: document.documentElement.outerHTML.replace(/&amp;/gm, '&'),
```

Comment:
jsdom encodes the html content but since our tests were failing, I had to replace back to `&` in the template

Reply:
Ohh, where are we even using an ampersand in our emails?

I think we want to leave character encoding in-place. It's OK if the tests have to check for `&amp` instead of `&`.